netty/src/main/java/io/grpc/netty/NettyClientHandler.java

```diff
@@ -149,7 +149,7 @@ static NettyClientHandler newHandler(
     Http2FrameWriter frameWriter = new DefaultHttp2FrameWriter();
     Http2Connection connection = new DefaultHttp2Connection(false);
     WeightedFairQueueByteDistributor dist = new WeightedFairQueueByteDistributor(connection);
-    dist.allocationQuantum(16 * 1024); // Make benchmarks fast again.
+    dist.allocationQuantum(getDistributorAllocationQuantum()); // Make benchmarks fast again.
```

Comment:
I think the comment "Make benchmarks fast again." was associated with the value `16 * 1024` so it might not apply with this code